pyramid_openapi3/tween.py

```diff
@@ -0,0 +1,53 @@
+"""A tween to validate openapi responses."""
+from .exceptions import ResponseValidationError
+from .wrappers import PyramidOpenAPIRequest
+from .wrappers import PyramidOpenAPIResponse
+from pyramid.httpexceptions import HTTPNotFound
+from pyramid.request import Request
+from pyramid.response import Response
+from pyramid.tweens import reraise
+
+import sys
+import typing as t
+
+
+def response_tween_factory(handler, registry) -> t.Callable[[Request], Response]:
+    """Create response validation tween.
+
+    This tween should run after pyramid exception renderer view, so that
+    final response status and content_type are known and can be validated.
+
+    The only problem here is, that when response validation fails, we have
+    to return some exception response, with an unknown content type.
+    The advantage is, that this are server errors, and if 500 errors are
+    only possibly due to response validation errors we don't need to document
```

Comment:
s/possibly/possible

Reply:
good catch ... probably also s/this/these ... I'll update